sdk/cosmos/azure-cosmos/azure/cosmos/aio/_cosmos_client.py

```diff
@@ -250,6 +251,32 @@ def from_connection_string(
             **kwargs
         )
 
+    @overload
+    async def create_database(
+            self,
+            id: str,
+            *,
+            offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
+            initial_headers: Optional[Dict[str, str]] = None,
+            throughput_bucket: Optional[int] = None,
+            return_headers: Optional[bool] = False,
+            **kwargs: Any
+    ) -> DatabaseProxy:
+        ...
+
+    @overload
+    async def create_database(
+            self,
+            id: str,
+            *,
+            offer_throughput: Optional[Union[int, ThroughputProperties]] = None,
+            initial_headers: Optional[Dict[str, str]] = None,
+            throughput_bucket: Optional[int] = None,
+            return_headers: Optional[bool] = True,
```

Comment:
```suggestion
            return_headers: Literal[True],
```
This can't have a default, because the default is already `False` - so to make it true, requires a user to provide a value (i.e. it becomes a required parameter). Does that make sense?